src/js/components/Markdown/README.md

```diff
@@ -34,4 +30,4 @@ You can see all the options [here](https://github.com/probablyup/markdown-to-jsx
 
 ```
 div
-```
+```
```

Comment:
This file is fixed on https://github.com/grommet/grommet/pull/4043,  I'll merge with master once PR #4043 will be merged.